backend/game/cards/keep_cards/victory_points_manipulation_cards/eater_of_the_dead.py

```diff
@@ -0,0 +1,13 @@
+from game.cards.keep_card import KeepCard
+from game.cards.card import Card
+from game.values import constants
+
+
+class EaterOfTheDead(KeepCard):
+    def __init__(self):
+        Card.__init__(self, "Eater of the Dead", 4, "+ 3[Star] every time a monster's [Heart] goes to 0.")
+
+    def special_effect(self, player_that_bought_the_card, other_players):
+        for other_player in other_players:
+            if other_player.current_health == constants.DEATH_HIT_POINT:
+                player_that_bought_the_card.update_victory_points_by(3)
```

Comment:
What triggers special_effect?
I don't think this code works.
When player 1 dies, this special effect kicks off and gains them 3 VP
When player 2 dies, this special effect kicks off and gains them 6 VP, because 2 players are now dead.

I believe this for/if doesn't work, and instead this method should just be "increase VP by 3" and then the hard part is only calling this method when a player dies.

Reply:
Maybe these cards are responsible for their own state?  For example, this one could keep a list of the players who died.  Yes there may be logic that might be shared across cards but we can factor them out over time if we treat each individual one as it own entity?  Just a possible thought, curious on @borisv13 opinion in regards to this